docs/UserGuide.md

```diff
@@ -254,6 +254,45 @@ Here is an example with steps to follow:
 * You can only **find assignments with keywords of the same prefix**. For example, `find n/Assignment d/23-10-2020` will not work.
 </div>
 
+### Editing your assignment : `edit`
+
+Format: `edit INDEX PREFIX/EDITTED_FIELD`
+
+You can specify an assignment's index to edit its fields. These fields are the name, module code and deadline of assignment.
```

Comment:
You may want to consider mentioning the user benefit of the edit command as well :)

Reply:
amended!